docs/protocol.md

```diff
@@ -46,6 +50,11 @@ TODO
 #### AppendRequest
 ```
 topic data
++--------+----------------+--------------+
+|   4    |       N        |  4 + N -> M  |
++--------+----------------+--------------+
+|  len   |     topic      |   Data...    |
++--------+----------------+--------------+
```

Comment:
APPEND data section has it's own length prefix independent of the overall message?

Reply:
Ahh, I see. But data could start with '/' so long as the client explicitly set the topic. For example:

`APPEND / /foo`

Should work